doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -302,6 +301,7 @@ The rules are:
 
 * Allowed characters are the ones in the plain ASCII set except the following ones: whitespace, `&`, `?`, `/` and `#`.
 * Maximum field length is 256 characters.
+* Minimum field length is 1 character.
```

Comment:
This new restriction has not been yet implemented. I have created this issue for not forgetting about it: https://github.com/telefonicaid/fiware-orion/issues/1836

NTC